gui/canvaspainter/v7/cef/cef_main.cxx

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2013 The Chromium Embedded Framework Authors. All rights
+// reserved. Use of this source code is governed by a BSD-style license that
+// can be found in the LICENSE file.
```

Comment:
I add CEF license file - here we are using cef_simple example practically without any change